### DISCUSSION TITLE
is it stalled? or still in development?

### DISCUSSION THREAD

**Discussion by Arvrairobo** [polonel/trudesk#626] [Q&A] [answered]
Hello trudesk team, this is really very good software, is it still in development or it is stalled?

what is the license for this software? open source, free to sell and distribute? please let me know more details on licensing as i can see copyright notice in that.

@polonel @shahyar 

Thanks guys.

**Comment by shahyar** [comment, marked as answer]
The current version is licensed as can be found here: https://github.com/polonel/trudesk/blob/master/LICENSE

Both the current and next versions are in ongoing, parallel development. Albeit at different paces.

**Comment by Arvrairobo** [comment]
ok got it, thank you for the kind answer, so if i want to present this to few of my customer, i can offer latest released version without any copyright or royalty to you? correct?

**Reply by shahyar** [reply]
I don't understand your question, In any case, we write software, not give legal advice, so if you want full clarification on this, you'll probably want to speak with a lawyer. With that said, I should probably point out that the Apache License 2.0 requires that you retain any and all copyright notices and disclaimers.
